.github/workflows/base-component-descriptor.yaml

```diff
@@ -26,6 +26,23 @@ on:
         required: false
         type: string
         default: SAP SE
+      component-references:
+        required: false
+        type: string
+        description: |
+          relative path (to repo root) to YAML/Json file containing definition of
+          component-references added to base-component-descriptor, defaults to
+          `.ocm/component-references.yaml`.
+          action will only raise on absence of file if an explicit value is given.
+
+          Example contents:
+          - componentName: github.com/gardener/gardener
+            version: v1.110.0
+            name: gardener/gardener
```

Comment:
I think it is a great idea to add an example, however consider:
- putting it as a code-block (\`\`\`...\`\`\`)
- maybe add a comment indicating the filepath (to make the example even easier to understand/follow)
- `/` character is not allowed in cref-names as of OCM-spec (I think)